js/index.js

```diff
@@ -1,9 +1,13 @@
+/* global L */
+/* global anime */
+
+
 import { SlideDeck } from './slidedeck.js';
 
 const map = L.map('map', {scrollWheelZoom: false}).setView([0, 0], 0);
 
 // ## The Base Tile Layer
-const baseTileLayer = L.tileLayer('https://tiles.stadiamaps.com/tiles/stamen_watercolor/{z}/{x}/{y}.jpg', {
+const baseTileLayer = L.tileLayer('https://tiles.stadiamaps.com/tiles/alidade_satellite/{z}/{x}/{y}.jpg', {
```

Comment:
**suggestion:** To use these tiles you might have to sign up for a Stadia account. The tiles will work on your local computer, because Stadia allows you to use tiles for development purposes. However, if you want to use these same tiles in a live site, you'll have to get an API key. Otherwise you'll see something like this:
![image](https://github.com/user-attachments/assets/5bcf2283-0069-493a-b317-ef1473314564)

The service is likely free up to some relatively high limit. See https://docs.stadiamaps.com/tutorials/raster-maps-with-leaflet/#authenticating-requests-via-api-key for instructions, and let me know if you have any trouble.